setup.sh

```diff
@@ -23,8 +23,8 @@ fi
 # Check out the pinned libvpx version.
 pushd libvpx
 git fetch
-git checkout --detach e758f9d45704ea0247c1b654f8602c967fa44199
+git checkout --detach master
```

Comment:
Hi, the previous tagged commit is intentional, that way two checkouts get the same version of the library they're comparing. Unfortunately that hash is ancient since I stopped working on this.

If you want to you can update the hash to the current master (so that you can remove the spatial-svc flag below). You can also add a --revision command-line flag so that you could run ./setup.sh --revision=master.